packages/app-store/stripepayment/components/EventTypeAppCardInterface.tsx

```diff
@@ -33,6 +33,15 @@ const EventTypeAppCard: EventTypeAppCardComponent = function EventTypeAppCard({
   const paymentOptionSelectValue = paymentOptions.find((option) => paymentOption === option.value);
   const [requirePayment, setRequirePayment] = useState(getAppData("enabled"));
 
+  let credentialId: number;
+
+  if (eventType.teamId) {
```

Comment:
I'm noticing a pattern here

Reply:
How can we be more DRY in here?

Reply:
Fixed in this commit https://github.com/calcom/cal.com/commit/b88334b0cd6c8b92ba7d2870c61e8797eb7ecbd3